### ISSUE TITLE
Client breaking change

### ISSUE THREAD

**Issue by wimski** [swisnl/php-http-fixture-client#29]
`php-http/mock-client` introduced a breaking change in a minor release. This breaks `Swis\Http\Fixture\Client::doSendRequest` because the parent method no longer exists.

https://github.com/php-http/mock-client/blame/a797c2a9122cccafcce14773b8a24d2808a9ab44/src/Client.php#L87

Adding this new method would fix it:
```php
/**
 * {@inheritdoc}
 */
public function sendRequest(RequestInterface $request): ResponseInterface
{
    $this->setDefaultResponse($this->fixtureResponseBuilder->build($request));

    return parent::sendRequest($request);
}
```

**Comment by JaZo** [comment]
Fixed in [2.3.1](https://github.com/swisnl/php-http-fixture-client/releases/tag/2.3.1)!

**Comment by JaZo** [comment]
Sorry, 2.3.1 is a bad release, please use [2.3.2](https://github.com/swisnl/php-http-fixture-client/releases/tag/2.3.2) instead! Monday mornings...
